### PR TITLE
Reorder script search bar

### PR DESCRIPTION
**Pull request by donggyu04**
Reorder script search bar.

1. Before

image

2. After

image

### CODE REVIEW

File: ngrinder-frontend/src/js/main.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import vuescroll from 'vuescroll';
 import numeral from 'numeral';
 import store from 'store/vuex-store.js';
 import VueRouterReferer from 'vue-router-referer.js';
+import VueClipboard from 'vue-clipboard2';
```
Comment (donggyu04) [inline ngrinder-frontend/src/js/main.js:15]: module is already included.

https://github.com/naver/ngrinder/blob/develop/ngrinder-frontend/package.json#L52

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 2]
```diff
@@ -35,6 +35,8 @@ common.message.validate.positiveNumber=\u8BF7\u8F93\u5165\u6B63\u6570\u3002
 common.message.alert.save.success=\u4FDD\u5B58\u6587\u4EF6\u6210\u529F\u3002
 common.message.alert.save.error=\u4FDD\u5B58\u6587\u4EF6\u5931\u8D25\u3002
 common.message.loading.error=
+common.message.copy.success=
+common.message.copy.fail=
```
Comment (junoyoon) [inline ngrinder-controller/src/main/resources/messages_cn.properties:39]: what about adding this translation?

Comment (junoyoon) [reply ngrinder-controller/src/main/resources/messages_cn.properties:39]: common.message.loading.error=加载{{content}}数据时出错
common.message.copy.success=复制成功
common.message.copy.fail=复制失败

Comment (donggyu04) [reply ngrinder-controller/src/main/resources/messages_cn.properties]: @junoyoon Thank you https://github.com/naver/ngrinder/pull/610/commits/090aa279ddb2eb577861fb5f9664b4401c40e590#diff-3b7a0e02f3c89e255ac112e1f9eb9704R37

**Pull request merged**